labels.mss

```diff
@@ -71,11 +70,17 @@
   [type='embassy'],[type='capital'] {
     shield-fill: @city_text;
     shield-face-name: @bold;
-    shield-size: 12;
+    shield-size: 13;
   }
   [type='embassy'] {
     shield-allow-overlap: true;
   }
+  [type='intermediate'] {
+    shield-face-name: @medium;
```

Comment:
Humm, why aren't those statements at the root?

Reply:
By default, town, it's regular

Intermediate, is at the moment a small subset of cities, isn't it